### ISSUE TITLE
link on readme broken

### ISSUE THREAD

**Issue by gieses** [Raschka-research-group/coral_pytorch#19]
FYI: The link at the end of the paragraph in the readme seems broken:

`This repository implements the CORAL and CORN functionality (neural network layer, loss function, and dataset utilities) for convenient use. Examples are provided via the "Tutorials" that can be found on the documentation website at https://raschka-research-group.github.io/coral_pytorch.`

**Comment by rasbt** [comment]
Thanks for the note. It should have been https://raschka-research-group.github.io/coral-pytorch with a dash, not underscore. Just fixed this
